core/modules/system/config/system.core.json

```diff
@@ -91,6 +91,7 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_login_page_settings": "tabs",
```

Comment:
There's currently no update hook, yet, to add that item (with default "tabs") to existing system.core.json config.